dev-python/dukpy/dukpy-0.3.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="JavaScript in Python"
+DESCRIPTION="dukpy is a JavaScript runtime environment for Python (tested with python 2.7 and >= 3.4) using the duktape embeddable JavaScript engine. With dukpy, you can run JavaScript in Python."
+HOMEPAGE="https://github.com/kovidgoyal/dukpyt"
+COPYRIGHT="2007-2019 Ian Bicking and contributors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/kovidgoyal/dukpy/archive/v0.3.tar.gz"
+CHECKSUM_SHA256="cc78c39ebba51f381c623b164cfb7dcf3caddf515fe7094bc53b7eca5d4e435e"
+
+
+ARCHITECTURES="any"
```

Comment:
As said on IRC, architecture can't be any here, but seeing you focus on x86_64 try to change this to:
```diff
-ARCHITECTURES="any"
+ARCHITECTURES="!x86_gcc2 x86_64"
```

Reply:
Would be nice if you could check this on gcc2h (32bit) and fix it)

Reply:
ok workes on 64